tests/lora/utils.py

```diff
@@ -212,10 +218,10 @@ def test_simple_inference(self):
             pipe.set_progress_bar_config(disable=None)
 
             _, _, inputs = self.get_dummy_inputs()
-            output_no_lora = pipe(**inputs).images
+            output_no_lora = pipe(**inputs)[0]
             self.assertTrue(output_no_lora.shape == self.output_shape)
 
-    def test_simple_inference_with_text_lora(self):
+    def test_simple_inference_with_text_lora(self, expected_atol: float = 1e-3, expected_rtol: float = 1e-3):
```

Comment:
For Cog, we do see the differences but need a lower tolerance for the tests to pass, so I've parameterized these